scripts/dapps.js

```diff
@@ -131,7 +131,7 @@ const logServing = () => {
   output.push(`\nbuild type           : ${watching ? 'serving' : 'building'}`);
   output.push(`environment          : ${nodeEnv}`);
   if (expressApp) {
-    output.push('started local server : http://localhost:3000');
+    output.push('started local server : http://localhost:3000/dev.html');
```

Comment:
Dev.html does not exist anymore in the latest `ui-dapp-browser` version
```suggestion
    output.push('started local server : http://localhost:3000');
```